ArrayList.java

```diff
@@ -0,0 +1,39 @@
+
+public class ArrayList<T> {
+    public Object[] arrayList;
```

Comment:
```suggestion
public class ArrayList<T> {

    public Object[] arrayList;
```
[formatting] run a formatter.